gallery/src/components/NxToast/NxToastFullPageExample.tsx

```diff
@@ -24,6 +24,9 @@ import {
   NxTile,
   NxP
 } from '@sonatype/react-shared-components';
+
+import {ToastModel} from '@sonatype/react-shared-components/components/NxToast/types';
+
```

Comment:
I'm struggling with proper implementation of typescript in the gallery. When defining the state for the toasts array, an error is thrown of type toasts: never[] (unless type ToastModel is defined). Is there a cleaner export method within the library I am missing that can prevent this explicit import of { ToatsModel } in the gallery example?

Reply:
> I'm struggling with proper implementation of typescript in the gallery. When defining the state for the toasts array, an error is thrown of type toasts: never[] (unless type ToastModel is defined). Is there a cleaner export method within the library I am missing that can prevent this explicit import of { ToatsModel } in the gallery example?

The "ToastModel" used by this example is entirely the business of this example, and not of the RSC library (unless I'm missing something).  You should define that type right in this example instead of in the `NxToast` types file